apps/stories/management/commands/create_user_accounts.py

```diff
@@ -69,9 +68,10 @@ def handle(self, *args, **options):
             block = row[1].strip().lower()
             cluster = row[2].strip().lower()
             group_name = row[3].strip()
-
+         
+            
```

Comment:
Please don't have two blank lines between code. Two blank lines are used between two class definitions usually. Going through pep8 recommendations should have more information.